sources/packages/web/src/views/student/StudentProfileEdit.vue

```diff
@@ -54,6 +54,7 @@ export default {
         dateOfBirth: studentInfo.birthDateFormatted,
         mode: StudentProfileFormModes.StudentEdit,
         identityProvider: AuthService.shared.userToken?.IDP,
+        sinConsent: false,
```

Comment:
I was not able to follow, maybe I am missing something, why this value is not coming from DB?